src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs

```diff
@@ -16,15 +16,10 @@
 
 using System;
 using System.Diagnostics;
-using System.Linq;
-#if NET461 || NETSTANDARD2_0
-using OpenTelemetry.Logs;
-#endif
-using OpenTelemetry.Resources;
 
 namespace OpenTelemetry.Exporter
 {
-    public class ConsoleExporter<T> : BaseExporter<T>
+    public abstract class ConsoleExporter<T> : BaseExporter<T>
```

Comment:
Good practice to make ctor `protected` on abstract class since it can't be "newed" directly.

Reply:
Fixed this.